js-exercises/deep-copy-object/deepCopyObject.js

```diff
@@ -0,0 +1,16 @@
+const deepCopyObject = (objToCopy) => {
+  if (objToCopy == null || typeof objToCopy !== 'object') {
+    return objToCopy;
+  }
+  const clonedObj = {};
+  for (const key in objToCopy) {
+    if (typeof objToCopy[key] === 'object' && objToCopy[key] !== null) {
```

Comment:
@Rabeeh167 do we really need this check ?. It seems redundant to me.

Reply:
Hi @satyambnsal, Thanks for your review.
I understand that we don't need this additional check. Please review my updated PR.